presto-main/src/test/java/io/prestosql/operator/TestWindowOperator.java

```diff
@@ -727,6 +731,67 @@ public void testFindEndPosition()
         assertFindEndPosition("000000000000001111111111", 14);
     }
 
+    @Test(dataProvider = "spillEnabled")
+    public void testDistinctPartitionAndPeers(boolean spillEnabled, boolean revokeMemoryWhenAddingPages, long memoryLimit)
```

Comment:
please pick up https://github.com/trinodb/trino/pull/6462 as a prefix commit

Reply:
Sorry, this might be kind of n00b, but what does it mean to pick up a prefix commit?

Reply:
I mean to cherry pick it before other commits

Reply:
You mean integrate into this PR chain right?

Reply:
Yup. Or rebase after I merge that PR